Mathlib/SetTheory/Cardinal/Ordinal.lean

```diff
@@ -573,12 +579,12 @@ theorem mul_aleph0_eq {a : Cardinal} (ha : ℵ₀ ≤ a) : a * ℵ₀ = a :=
   (mul_eq_max ha le_rfl).trans (max_eq_left ha)
 #align cardinal.mul_aleph_0_eq Cardinal.mul_aleph0_eq
 
--- Porting note (#10618): removed `simp`, `simp` can prove it
+-- Porting note: removed `simp`, `simp` can prove it
```

Comment:
Why are dropping this classification?

Reply:
Good catch; this was a rebase error - I just reverted this change.